model-optimizer/extensions/front/onnx/resize_to_interpolate.py

```diff
@@ -77,24 +78,30 @@ def pattern(self):
 
     def replace_sub_graph(self, graph: Graph, match: dict):
         resize_node = match['resize']
+        resize_node_name = resize_node.name
```

Comment:
Change to `resize_node_name = resize_node.soft_get('name', resize_node.id)`

Reply:
Fixed.